test/organizations/OrganizationCompletionEntity.js

```diff
@@ -0,0 +1,196 @@
+import { OrganizationEntity } from '../../src/organizations/OrganizationEntity.js';
+import { testData } from './data/OrganizationCompletionEntity.js';
+
+describe('OrganizationEntityCompletions', () => {
+	let trackingNoDisplay, trackingAndDisplay, noTrackingDisplay, noActions;
+
+	beforeEach(() => {
+		const trackingNoDisplayJson = window.D2L.Hypermedia.Siren.Parse(testData.trackingNoDisplay);
+		trackingNoDisplay = new OrganizationEntity(trackingNoDisplayJson);
+
+		const trackingAndDisplayJson = window.D2L.Hypermedia.Siren.Parse(testData.trackingAndDisplay);
+		trackingAndDisplay = new OrganizationEntity(trackingAndDisplayJson);
+
+		const noTrackingJson = window.D2L.Hypermedia.Siren.Parse(testData.noTracking);
+		noTrackingDisplay = new OrganizationEntity(noTrackingJson);
+
+		const noActionsJson = window.D2L.Hypermedia.Siren.Parse(testData.noActions);
+		noActions = new OrganizationEntity(noActionsJson);
+	});
+	describe('properties working', () => {
+		it('tracking, no display', () => {
+			expect(trackingNoDisplay.isCompletionTracked).to.be.true;
```

Comment:
ideally unittest should assert behavior of one independend method.
If `trackingNoDisplay.isCompletionTracked` breaks, the test will not verify `trackingNoDisplay.isProgressDisplayed`